doto-backend/test/task.test.js

```diff
@@ -0,0 +1,157 @@
+'use strict';
+
+const mongoose = require('mongoose');
+const TaskModel = require('../src/models/Task');
+const UserModel = require('../src/models/User');
+const assert = require('assert');
+
+const validUser = new UserModel({
+    name: 'john',
+    picture: 'profile.png',
+    themePreference: 'dark'
+})
+
+process.env.TEST_SUITE = 'task-test';
+
+describe('Task Model Tests', () => {
+
+    before(async () => {
+        await mongoose.connect(`mongodb://localhost:27017/${process.env.TEST_SUITE}`, { useNewUrlParser: true}, (err) => {
```

Comment:
hey could you please change the url to this
`mongodb://127.0.0.1:27017/${process.env.TEST_SUITE}`